privacy/index.php

```diff
@@ -1,24 +1,3 @@
 <?php
-  require_once('includes/template.php');
-  
-  // Required
-  head([
-    'title' =>'Privacy Policy | Keyman',
-    'css' => ['template.css'],
-    'showMenu' => true
-  ]);           
+header('location: https://software.sil.org/language-software-privacy-policy/');
```

Comment:
Can we turn this into a web.config please? Trying to eliminate use of pure redirects in PHP for future maintenance.